web/libs/editor/tests/e2e/tests/rich-text/rich-text-edge-cases.test.js

```diff
@@ -15,7 +15,7 @@ const edgeCaseConfig = `<View>
     <HyperText name="html" value="$html" />
 </View>`;
 const edgeCaseHtml = `<p>This
-is an <u><span>e</span><span>x</span><span>a</span><span>m</span><span>p</span><span>l</span><span>e</span></u><br/> of 
+is an <u><span>e</span><span>x</span><span>a</span><span>m</span><span>p</span><span>l</span><span>e</span></u><br/> of
```

Comment:
That linting keeps killing test data.
```suggestion
is an <u><span>e</span><span>x</span><span>a</span><span>m</span><span>p</span><span>l</span><span>e</span></u><br/> of 
```